exercises/dominoes/test.ml

```diff
@@ -4,24 +4,30 @@ open Dominoes
 
 let print_dominoe (d1, d2) = sprintf "(%d,%d)" d1 d2
 
+let dominoes_printer xs = String.concat ~sep:";" (List.map xs ~f:print_dominoe)
 let option_printer = function
   | None -> "None"
-  | Some xs -> "Some [" ^ String.concat ~sep:";" (List.map xs ~f:print_dominoe) ^ "]"
+  | Some xs -> "Some [" ^ dominoes_printer xs ^ "]"
 
 let drop_1_right xs = List.rev xs |> List.tl_exn |> List.rev
 
-let check_chain (chained: dominoe list) = 
+let rotate (x, y) = if x > y then (y, x) else (x, y)
+let norm l = List.map ~f:rotate l |> List.sort ~cmp:compare
+
+let check_chain (input: dominoe list) (chained: dominoe list) =
   let assert_dominoes_match d1 d2 =
     if snd d1 <> fst d2 then failwith @@ sprintf "%s and %s cannot be chained together" (print_dominoe d1) (print_dominoe d2) else () in
   let consecutives = List.zip_exn (drop_1_right chained) (List.tl_exn chained) in
```

Comment:
or maybe I can change this to `List.zip_exn chained (List.tl_exn chained ++ [List.hd_exn chained])`, so that the list of consecutives will include `(List.last_exn chained, List.hd_exn chained)`